datacenter/ucp/2.2/guides/admin/install/architecture-specific-images.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Architecture-specific images
+description: Learn how to use images that are specific to particular hardware architectures in Docker Universal Control Plane.
+keywords: UCP, Docker EE, image, IBM z
+---
+
+Docker Universal Control Plane deploys images for a number of different
+hardware architectures, including IBM z systems. Some architectures
+require pulling images that have specific tags indicating the target
+architecture.
+
+## Tag for IBM z Systems
```

Comment:
We also have `docker/ucp-agent-win` and `docker/ucp-dsinfo-win` so it might make sense to talk about Windows here too.

It also might make sense to add a note to `/architecture#ucp-internal-components` telling that the name for the component will actually depend on the node's OS.